content/en/docs/develop/core-modules/wasm.md

```diff
@@ -142,9 +156,18 @@ Uploads new code to the blockchain and results in a new code ID, if successful.
 
 ```go
 type MsgStoreCode struct {
-	Sender sdk.AccAddress `json:"sender" yaml:"sender"`
+	// Sender is the that actor that signed the messages
```

Comment:
plz remove all `that` prior to `actor`

Reply:
https://github.com/xpladev/docs/pull/10/commits/07c7f7870d2891a3dbc8059de5f50b835300f417